### PR TITLE
Adds Test for measureTable component

### PR DESCRIPTION
**Pull request by kagehina919**
- [ ] TODO: Update code to handle #335 and add tests for it.

@wlach I have written one test for the component `measureTable`. How should we get the `sampleMeasures` and `sampleVersions` for testing? I have used one sample value but how I can use the actions and reducers to fetch it inside the test?

### CODE REVIEW

**Comment by wlach** [comment]
I think you are supposed to commit snapshots to the repository -- that way we'll notice if they ever change. :) https://jestjs.io/docs/en/snapshot-testing

**Comment by kagehina919** [comment]
@wlach added `__snapshots__` directory. Please review.

**Comment by kagehina919** [comment]
@wlach Added test to check `it should not render the measures table when measures is an empty list`. Please review. Fixes #335.

File: frontend/ui/measuretable.jsx [hunk 1]
```diff
@@ -25,6 +25,10 @@ const getReleaseValue = (release, countType, timeWindow) => {
 
 class MeasureTable extends React.Component {
   render() {
+    if (this.props.measures.length === 0) {
```
Comment (wlach) [inline frontend/ui/measuretable.jsx]: Returning `null` to not display something is kind of counterintuitive. I actually think the conditional here should be on what's rendering `MeasureTable` (subview in this case), probably around here: https://github.com/mozilla/missioncontrol/blob/5daea6f89ed5b780aa553b0cffdae06857a9341f/frontend/ui/subview.jsx#L166

Could you back this out (along with the test)? Perhaps we can write some tests for subview when modifying that...

Comment (kagehina919) [reply frontend/ui/measuretable.jsx]: I have added the check in `subview.jsx`. I'll add the test for this along with other tests for subview in a later PR!

Comment (kagehina919) [reply frontend/ui/measuretable.jsx]: @wlach fixed. I'll open another PR for handling #335. Please review.

**Comment by wlach** [comment]
Looks great now, thank you!

**Pull request merged**